website/src/reference/builder/member/getter.md

```diff
@@ -0,0 +1,87 @@
+# `overwritable` :microscope:
```

Comment:
```suggestion
# `getter` :microscope:
```

Let's also update the table in refererce/builder.md.

You also need to update the sidebar config in `website/.vitepress/config.mts`. Preserve the lexicographical sorting when adding an entry there.